docs/static/security/logstash.asciidoc

```diff
@@ -5,10 +5,10 @@
 <titleabbrev>Security</titleabbrev>
 ++++
 
-The Logstash {es} plugins ({logstash-ref}/plugins-outputs-elasticsearch.html[output],
-{logstash-ref}/plugins-inputs-elasticsearch.html[input],
-{logstash-ref}/plugins-filters-elasticsearch.html[filter]
-and {logstash-ref}/monitoring-logstash.html[monitoring])
+The Logstash {es} {logstash-ref}/plugins-outputs-elasticsearch.html[output],
+{logstash-ref}/plugins-inputs-elasticsearch.html[input], and
+{logstash-ref}/plugins-filters-elasticsearch.html[filter] plugins,  as well as
+{logstash-ref}/monitoring-logstash.html[monitoring] and central management,
 support authentication and encryption over HTTPS.
 
 To use Logstash with a secured cluster, you need to configure authentication
```

Comment:
Since the cluster will be secured by default, we should make this more explicit.

"Starting in version 8.0, Elasticsearch clusters are secured by default. Therefore it is necessary to configure authentication ... "